NomaiGrandPrix/NomaiGrandPrix.cs

```diff
@@ -87,6 +87,8 @@ private void Update()
                 HandleNewLoopSetup();
                 InitMapMarker();
                 SpawnGoal(_goalPoint.transform);
+                var spawnActions = SpawnActionFactory.GetActionsForSpawn(SpeedrunState.SpawnPoint?.internalId);
```

Comment:
In what situation would `SpeedrunState.SpawnPoint` be null? Is that before it's set by the player or something?

Reply:
yes

Reply:
Interesting. I bit there's a more type-safe way of structuring that data to avoid this null-check; might poke around at it later. Obviously not a blocker.